### PR TITLE
Admin permissions now synced with environment variables. 

### PR DESCRIPTION
**Pull request by Szsaunders**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,8 @@
     "rxjs": "~6.5.2",
     "rxjs-compat": "^6.5.2",
     "tslib": "^1.10.0",
+    "update": "^0.7.4",
+    "webdriver-manager": "^12.1.5",
```
Comment (escalonn) [inline package.json]: why are these dependencies added

File: src/app/components/navbar/navbar.component.ts [hunk 2]
```diff
@@ -19,13 +20,13 @@ import { AuthService } from '@revaturexyz/profile';
 export class NavbarComponent implements OnInit, AfterViewInit {
 
   // public LoggedIn = this.authService.isLoggedIn;
-  public LoggedIn = false;
+  public LoggedIn = !environment.requireAuth;
```
Comment (escalonn) [inline src/app/components/navbar/navbar.component.ts]: can we name this variable something more sensible

File: package.json (hunk 1 above)
Comment (Szsaunders) [reply package.json]: e2e tests were giving me trouble, so I tried to run the command locally to see what was going wrong. I'll get rid of those.

**Comment by escalonn** [comment]
also clicking on profile breaks.

File: package.json (hunk 1 above)
Comment (escalonn) [reply package.json]: `ng e2e` should work fine locally without any extra deps.

File: src/app/components/navbar/navbar.component.spec.ts [hunk 3]
```diff
@@ -54,12 +54,18 @@ describe('NavbarComponent', () => {
...
     });
     component.login();
   });
+  xit('should be logged in and have have the correct user after successful login', () => {
+    inject([MsalService], () => {
+      expect(component.loggedIn).toBe(false);
+      expect(component.isAdmin).toBe(false);
+    });
+    component.logout();
+  });
```
Comment (escalonn) [inline src/app/components/navbar/navbar.component.spec.ts]: why are you adding a test which you are excluding from running (`xit`) whose description isnt correct?

File: src/app/modules/routing/routing.module.ts [hunk 4]
```diff
@@ -15,9 +15,9 @@ import { MsalGuard } from '@azure/msal-angular';
  */
 const appRoutes: Routes = [
   { path: '', component: MainImageComponent },
-  { path: 'forecast', component: ForecastComponent, canActivate: [MsalGuard] },
-  { path: 'selection', component: SelectionComponent, canActivate: [MsalGuard] },
-  { path: 'profile', component: ProfileComponent, canActivate: [MsalGuard] },
+  { path: 'forecast', component: ForecastComponent },
+  { path: 'selection', component: SelectionComponent},
+  { path: 'profile', component: ProfileComponent},
```
Comment (escalonn) [inline src/app/modules/routing/routing.module.ts]: really you should make a new guard to wrap the msalguard, but for now just put a user story on the board for it.

File: src/app/components/navbar/navbar.component.spec.ts (hunk 3 above)
Comment (Szsaunders) [reply src/app/components/navbar/navbar.component.spec.ts]: There is a very good reason for that. That reason is that I'm tired.

**Pull request merged**